README.md

```diff
@@ -123,11 +123,7 @@ var rtm = new RtmClient(token, {
   // Sets the level of logging we require
   logLevel: 'error',
   // Initialise a data store for our client, this will load additional helper functions for the storing and retrieval of data
-  dataStore: new MemoryDataStore(),
-  // Boolean indicating whether Slack should automatically reconnect after an error response
-  autoReconnect: true,
```

Comment:
The option is certainly valid, however it defaults to `true`. Therefore, having `autoReconnect: true` is redundant.

I could see value in demonstrating the existence of the option since this is an example. However, there are several other options not in the example which a user would need to find in the source code documentation. Thus, since the option is non-essential and redundant, I think it is most intuitive to consistently use only those options which are essential.